lib/contracts/formatters.rb

```diff
@@ -90,7 +90,7 @@ def full?
 
       def plain?
         # Not a type of contract that can have a custom to_s defined
-        !@value.is_a?(CallableClass) && @value.class != Class
+        !@value.is_a?(Builtin::CallableClass) && @value.class != Class
```

Comment:
This is required or all spec will fail
(`#<NameError: uninitialized constant Contracts::Formatters::InspectWrapper::CallableClass>`)